blockbase/inc/customizer/wp-customize-fonts.php

```diff
@@ -273,18 +287,40 @@ function initialize( $wp_customize ) {
 			)
 		);
 
-		$this->add_setting_and_control( $wp_customize, 'body', __( 'Body font', 'blockbase' ), $body_font_slug );
-		$this->add_setting_and_control( $wp_customize, 'heading', __( 'Heading font', 'blockbase' ), $heading_font_slug );
+		// Add a reset button
+		$this->font_control_default_body    = $body_font_default['slug'];
+		$this->font_control_default_heading = $heading_font_default['slug'];
+		$wp_customize->add_control(
```

Comment:
Can we make this only show when a non-default value is selected?

Reply:
yup

Reply:
Well MOSTLY.
It determines if they are set to default on startup.
If so, hide the button.

Any time the values changes, show the button.

Any time the values are reset to default, hide the button.

Only use case not covered is that if the values are set to the default values manually the button is still shown.  I didn't find an easy way to work around that and hope we can live with that limitation...